plugins/carbon_black_defense/help.md

```diff
@@ -93,131 +93,68 @@ Example input:
+  "contacted": 5,
+  "num_aggregated": 503,
+  "num_available": 500,
+  "num_found": 78750,
   "results": [
     {
-      "eventId": "1defe38112e911e7b34047d6447797bd",
-      "processDetails": {
-        "userName": "SYSTEM",
-        "processId": 2872,
-        "milisSinceProcessStart": 32,
-        "name": "taskeng.exe",
-        "parentPid": 772,
-        "interpreterHash": null,
-        "interpreterName": null,
-        "commandLine": "taskeng.exe {5267BC82-9B0D-4F0B-A566-E06CDE5602F1} S-1-5-18:NT AUTHORITY\\System:Service:",
-        "parentName": "svchost.exe",
-        "parentPrivatePid": "772-1489763380982-18",
-        "targetPid": 2468,
-        "targetPrivatePid": "2468-1490617768051-975",
-        "parentCommandLine": "C:\\Windows\\system32\\svchost.exe -k netsvcs",
-        "targetCommandLine": "C:\\Program Files (x86)\\Google\\Update\\GoogleUpdate.exe",
-        "privatePid": "2872-1490617768004-974",
-        "targetName": "GoogleUpdate.exe",
-        "fullUserName": "NT AUTHORITY\\SYSTEM"
-      },
-      "eventTime": 1490617768036,
-      "selectedApp": {
-        "applicationName": "taskeng.exe",
-        "virusName": null,
-        "reputationProperty": "TRUSTED_WHITE_LIST",
-        "effectiveReputation": null,
-        "applicationPath": "C:\\Windows\\System32\\taskeng.exe",
-        "md5Hash": "a21ac8d41e63cf1aa24ebc165ae82c9a",
-        "effectiveReputationSource": null,
-        "virusCategory": null,
-        "sha256Hash": "74b9cf472d5008e00735482f084f886eaa201248d6e87ab6b1990e3670bd6693",
-        "virusSubCategory": null
-      },
-      "attackStage": null,
-      "targetApp": {
-        "applicationName": "C:\\Program Files (x86)\\Google\\Update\\GoogleUpdate.exe",
-        "virusName": null,
-        "reputationProperty": "TRUSTED_WHITE_LIST",
-        "effectiveReputation": null,
-        "applicationPath": null,
-        "md5Hash": null,
-        "effectiveReputationSource": null,
-        "virusCategory": null,
-        "sha256Hash": "52fc3aa9f704300041e486e57fe863218e4cdf4c8eee05ca6b99a296efee5737",
-        "virusSubCategory": null
-      },
-      "registryValue": null,
-      "alertCategory": null,
-      "longDescription": "The application \"<share><link hash=\"74b9cf472d5008e00735482f084f886eaa201248d6e87ab6b1990e3670bd6693\">C:\\Windows\\System32\\taskeng.exe</link></share>\" attempted to invoke the application \"C:\\Program Files (x86)\\Google\\Update\\GoogleUpdate.exe\", by calling the function \"CreateProcessW\". The operation was successful.",
-      "threatIndicators": [
-        "SUSPENDED_PROCESS"
+      "backend_timestamp": "2022-03-15T10:28:11.423Z",
+      "device_group_id": 0,
+      "device_id": 4849627,
+      "device_name": "wb-auto-qa",
+      "device_policy_id": 6525,
+      "device_timestamp": "2022-03-15T10:26:39.253Z",
+      "enriched": true,
+      "enriched_event_type": "NETWORK",
+      "event_description": "The operation was <accent>blocked by Cb Defense</accent>.",
```

Comment:
Is the `<accent>` supposed to be in the example? I saw that it appeared in other event descriptions as well

Reply:
It is yes, descriptions come with some markup.